Src/Plugins/Security/Security.SharedStrings.txt

```diff
@@ -23,15 +23,23 @@
 
 # $FacebookAppCredentials=(?si)(((facebook|fb)(.{0,20})?[`'"\s>](?P<id>[0-9]{13,17})[^0-9].{1,500}?)|((facebook|fb)(.{0,20})?[`'"\s>](?P<secret>[0-9a-f]{32})([^0-9a-f]?)).{1,500}?){2}
 
-  $SEC101/001.HttpAuthorizationRequestHeader=(?i)(?P<scheme>http|ftp|https):\/\/(?P<host>[\w_.-]{1,200})(?P<path>[\w.,@?^=%&:\/~+#-]*[\w@?^=%&\/~+#-])?(.|\n){0,100}?authorization[,\[:= "']+(basic)[\s\r\n]{0,10}(?P<secret>[^'"><;\s]{1,500})
+# Intrafile analysis
+  $SEC101/004.FacebookAppCredentialsId=(?si)facebook.{1,50}\b(?P<id>[0-9]{15})([^0-9]|$)
+  $SEC101/004.FacebookAppCredentialsSecret=(?si)facebook.{1,50}\b(?P<secret>[0-9a-f]{32})([^0-9]|$)
+
+  $SEC101/008.AwsCredentialsId=(?s)\b(?P<id>(A3T[0-9A-Z]|AKIA|AGPA|AROA|AIPA|ANPA|ANVA|ASIA)[0-9A-Z]{16})([^0-9A-Z]|$)
```

Comment:
>\b [](http://example.com/codeflow?start=35&length=2)

we should identify a good prefix pattern to use here. \b will also match the beginning of a line and is a concise way to go. we could consider a more precise match, which requires a start line or a character which is not part of the match set (this is what we do for the end of the match). \b could be sufficient, though, and it's more readable. Thoughts?

Reply:
Agreed. simplifies the regex :)